infra/token-transfer-server/src/enums.js

```diff
@@ -32,6 +32,9 @@ const TransferStatuses = new Enum(
   'Expired'
 )
 
+const InvestorTypes = new Enum('Advisor', 'Strategic', 'CoinList')
```

Comment:
We'll need to update [that dude](https://github.com/OriginProtocol/origin/blob/master/infra/token-transfer-server/src/scripts/import_data.js#L155) but it can be done in a separate PR since anyway that guy probably needs a refresh.